src/rules/testing.ts

```diff
@@ -2,6 +2,7 @@ export = {
   plugins: ['chai-friendly'],
   rules: {
     'no-unused-expressions': 'off',
+    '@typescript-eslint/no-unused-expressions': 'off',
```

Comment:
When using `vaadin/javascript`, this rule is undefined. We need to figure out how to only override it when using `vaadin/typescript`.